services/README.md

```diff
@@ -0,0 +1,42 @@
+# Buddy App GraphQL service
+
+> Service that exposes GraphQL lambda to be consumed by Buddy App front end app. Connects also to a prisma db.
+
+## Prerequisites
+
+- Please assure that you have Node.js installed on your machine.
+
+- And one more `.env` file in the `/services` folder(the values should be changed to your own):
+
+```
+PRISMA_ENDPOINT=######
+PRISMA_SECRET=#######
+APP_SECRET==#######
+API_GATEWAY_DOMAIN=#######
+```
+
```

Comment:
I know, it was an issue that was happening in a `WIP` commit. It should be fixed at the moment.

Reply:
I added in `README` a section related to `prisma generate` in case of changes in the data model. 
See here https://github.com/BuildIt-Poland/buddy-app/pull/59/commits/2ed233f447d426f5aaacaab33e3a408259f11a2c